### DISCUSSION TITLE
joinp --ignore-case option help text

### DISCUSSION THREAD

**Discussion by datatraveller1** [dathere/qsv#2363] [General]
qsv 1.0.0:
`qsv joinp --help`
states "`However, joinp doesn't have an --ignore-case option.`"
But it seems to have one:
`joinp options:
    -i, --ignore-case      When set, joins are done case insensitively.`
Should the help text be corrected?

Futher, I have a suggestion: It would be nice if the following options could be added:
`--right-anti`
`--right-semi`

I know I can achieve this by using `--left-anti` and `--left-semi` and swapping `<columns1> <input1>` and `<columns2> <input2>`, but that would be more convenient.

**Comment by jqnatividad** [comment]
Created two issues from discussion:

https://github.com/dathere/qsv/issues/2376
https://github.com/dathere/qsv/issues/2377
